rbi/stdlib/uri.rbi

```diff
@@ -67,7 +350,7 @@ end
 #   class RSYNC < Generic
 #     DEFAULT_PORT = 873
 #   end
-#   @@schemes['RSYNC'] = RSYNC
+#   register_scheme 'RSYNC', RSYNC
```

Comment:
It looks like there are a handful of unrelated changes, likely due to this warning:

https://github.com/sorbet/sorbet/blob/72ca0f24493bd968088f6b9adf3134a23f28c105/rbi/tools/sync-rdoc.rb#L591-L595

Do you mind using `rbenv exec rbi/tools/sync-rdoc.rb ...` to use the same Ruby version as configured in `.ruby-version` for updating this file?

Reply:
(I've gone ahead and done this for the current PR for you.)